File: MPI/mpicheck/tests/functions/2.0/MPI_Win_get_errhandler.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Win var_0;
    MPI_Errhandler *var_1;
    int ret;
    /* calls */
    ret = MPI_Win_get_errhandler(var_0, var_1);
    ret = PMPI_Win_get_errhandler(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Win_get_errhandler.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_win_get_errhandler(var_0, var_1, var_2)
       call pmpi_win_get_errhandler(var_0, var_1, var_2)
        end program main
    